### ISSUE TITLE
Creating a Subscription Schedule returns an error but it actually creates it

### ISSUE THREAD

**Issue by CarlosTorrecillas** [stripe/stripe-dotnet#1808]
OS: Win 10
.NET Version: dotnet Core 2.2
Stripe.net version: 29.5.0

When I create a subscription schedule for a customer I get an error from the SubscriptionScheduleService however the actual subscription schedule is created.

Example:

```
var subscriptionScheduleCreateOptions = new SubscriptionScheduleCreateOptions
                {
                    CustomerId = customerId,
                    Phases = new List<SubscriptionSchedulePhaseOptions>
                    {
                        new SubscriptionSchedulePhaseOptions
                        {
                            Plans = new List<SubscriptionSchedulePhaseItemOptions>
                            {
                                new SubscriptionSchedulePhaseItemOptions
                                {
                                    PlanId = planId
                                }
                            },
                            DefaultPaymentMethodId = paymentMethodId,
                            EndDate = endDate
                        },
                    },
                    StartDate = startDate,
                    RenewalBehavior = "none"
                };
var subscriptionSchedule = await _subscriptionScheduleService.CreateAsync(subscriptionScheduleCreateOptions);
```
They are all valid variables and I can see the schedule create for a future date, with the corresponding period, etc. However the client returns the following:

```
Stripe.StripeException
  HResult=0x80131500
  Message=Invalid response object from API: "{
  "id": "sub_sched_1FRdzcGpwI99htqjPlQARbi5",
  "object": "subscription_schedule",
  "billing": "charge_automatically",
  "billing_thresholds": null,
  "canceled_at": null,
  "collection_method": "charge_automatically",
  "completed_at": null,
  "created": 1570623012,
  "current_phase": null,
  "customer": "cus_FxX4ZbcYuuXUmf",
  "default_payment_method": null,
  "default_source": null,
  "end_behavior": null,
  "invoice_settings": null,
  "livemode": false,
  "metadata": {
  },
  "phases": [
    {
      "application_fee_percent": null,
      "billing_thresholds": null,
      "collection_method": null,
      "coupon": null,
      "default_payment_method": "pm_1FRdD1GpwI99htqjMbXONphP",
      "default_tax_rates": [

      ],
      "end_date": 1594296599,
      "invoice_settings": null,
      "plans": [
        {
          "billing_thresholds": null,
          "plan": "plan_FxZ7jWGlemfjRD",
          "quantity": null,
          "tax_rates": [

          ]
        }
      ],
      "start_date": 1571227799,
      "tax_percent": null,
      "trial_end": null
    }
  ],
  "released_at": null,
  "released_subscription": null,
  "renewal_behavior": "none",
  "renewal_interval": null,
  "revision": "sub_sched_rev_1FRdzcGpwI99htqj6rhLxmZC",
  "status": "not_started",
  "subscription": null
}
"
  Source=Stripe.net
  StackTrace:
   at Stripe.StripeClient.ProcessResponse[T](StripeResponse response)
   at Stripe.StripeClient.<RequestAsync>d__25`1.MoveNext()
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Stripe.Service`1.<RequestAsync>d__23`1.MoveNext()
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
```
Is there anything you guys could look at?

Many thanks,
Carlos.

**Comment by remi-stripe** [comment]
@CarlosTorrecillas Thanks a lot for the report! I can reproduce so I'm working on a fix!

**Comment by CarlosTorrecillas** [comment]
Do you have an ETA of when you’ll get the fix out?

**Comment by remi-stripe** [comment]
@CarlosTorrecillas This has now been fixed in the latest version 30.0.1

**Comment by CarlosTorrecillas** [comment]
Thanks for this @remi-stripe . This now works as expected
